### PR TITLE
[AMS-764] import notes in advertisement moderation comment

### PR DESCRIPTION
**Pull request by xakep139**

### CODE REVIEW

File: src/MigrationTool/ImportService.cs [hunk 1]
```diff
@@ -587,6 +589,31 @@ where positions.Contains(pc.MasterPositionId)
...
+                                         .GroupBy(n => n.ParentId)
+                                         .ToDictionaryAsync(g => g.Key, g => g.Take(2).ToList());
+
+                foreach (var advertisement in advertisements)
+                {
+                    foreach (var element in advertisement.AdvertisementElements)
+                    {
+                        if (notes.ContainsKey(element.Id))
+                        {
+                            element.Notes = notes[element.Id];
```
Comment (denisivan0v) [inline src/MigrationTool/ImportService.cs]: `TryGetValue` method can be used instead of `ContainsKey` + indexer

Comment (xakep139) [reply src/MigrationTool/ImportService.cs]: Done in https://github.com/2gis/nuclear-vstore/pull/111/commits/34e2802a3181688ce401d41a54adf36a5d766d47

**Comment by xakep139** [comment]
@denisivan0v, take a look at https://github.com/2gis/nuclear-vstore/pull/111/commits/17e6b36a26d0292430c3302d94b954a8a1a95f22

**Comment by denisivan0v** [comment]
LGTM

**Pull request merged**